src/ts/main.ts

```diff
@@ -72,4 +72,4 @@ export { fromPerfCascadeFormat }
 export { transformHarToPerfCascade }
 export { makeLegend }
 // export typings
-export * from "./typing/index.d"
+export * from "./typing/index"
```

Comment:
all `*.ts` and `*.d.ts` changes are to make it interfaces and typings exportable via `run:tscEs6`